plugins.xml

```diff
@@ -2198,6 +2198,43 @@
 			</compatibility>
 			<certification type="partner"/><description>This plugin adds social media buttons to your web site without compromising the privacy of website users.</description>
 		</release>
+		<release date="2023-09-18" version="3.3.1.0" md5="de3b2530fe5aa1737ae62517e9fe5416">
```

Comment:
@ronste, is the version number wrong here? The archive filename says `3_3_0-2`, and the `version` attribute says `3.3.1.0`.

Reply:
> @ronste, is the version number wrong here? The archive filename says `3_3_0-2`, and the `version` attribute says `3.3.1.0`.

@asmecher , no, I can't see any issues there either. The version numbers seem to be ok. Historically the OJS version numbers have been used as release tags in this repo. Now that I have a second release within a major OJS version I also had to start numering. So filenames, versioning and tags now are:

shariff-3_3_0-1.tar.gz => plugin version 3.3.0.1 => 970b2ae418dec259c2e84e4364cc79cb

shariff-3_3_0-2.tar.gz => plugin version 3.3.1.0 => de3b2530fe5aa1737ae62517e9fe5416

shariff-3_4_0.tar.gz => plugin version 3.4.0.0 => 319d33932cc130e592768e6d9575f63a

shariff-3_4_0-1.tar.gz => plugin version 3.4.2.0 => 61db66ffb5b4398763d184165ae8a725

That's weird, I can't see any issue. There's a tabulator in excess but that's all.

Reply:
Ah, I think I was being thrown off by the version numbers, but that's a red herring. The plugin gallery toolset is only expecting one added plugin per pull request, and only gets the first MD5. I'll merge this as is and let it test fully, but next time, I'd suggest just adding one plugin release per PR.

Reply:
Ahh, I see. I'll try to remember this for the next PR. Thanks!